### ISSUE TITLE
Tray icon appears when exiting the quake shell

### ISSUE THREAD

**Issue by lhecker** [microsoft/terminal#16269]
### Windows Terminal version

1.19.2682.0

### Windows build number

10.0.22635.0

### Steps to reproduce

* Ensure the two notification area options are disabled
* Launch Windows Terminal
* <kbd>Win</kbd><kbd>`</kbd>
* Either
  * Execute `exit`
  * <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>W</kbd>

### Expected Behavior

No tray icon.

### Actual Behavior

A wild tray icon appears!

**Comment by vefatica** [comment]
There are two other phenomen associated with closing the quake window's shell.  In WT, press Win+backtick.  The quake window opens, nicely spanning the top of one monitor, its accent-color border right at the left, top, and right screen edges.  Exit the shell in the quake window and in the original WT, press Win+backtick again.  This time ...

1.  The quake window is wider; its accent border off the right and left screen edges, and
2.  It has a tab bar (but no tabs) at the top.

Here's the original WT with the left-top of the monitor (showing the second quake window) in the background.

![image](https://github.com/microsoft/terminal/assets/61856645/110e7b15-1321-44a1-9b78-0c75f821545a)

**Comment by vefatica** [comment]
Yes, I'm on Windows 10.  It could be the same issue; I didn't read carefully.  I saw something about "borderless" in there.  In my experiment, the quake window has a border (accent color) is both cases but it's on the monitor in the first case and off the monitor in the second case.
